frontend/packages/console-telemetry-plugin/console-extensions.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "type": "console.flag",
+    "properties": {
+      "handler": { "$codeRef": "telemetryFlags.detectTelemetry" }
+    }
+  },
+
+  {
+    "type": "console.telemetry/listener",
+    "properties": {
+      "listener": {
+        "$codeRef": "telemetry.eventListener"
```

I would recommend making it clear that this is an specific implementation

```suggestion
        "$codeRef": "telemetry.listeners.segment"
```